### DISCUSSION TITLE
Error: SQLITE_CANTOPEN: unable to open database file

### DISCUSSION THREAD

**Discussion by hagenderouen** [directus/directus#11207] [Archive]
I'm receiving the following error message when deploying to IIS:
`10:23:08 ⚠️  Directory for SQLite database file (%pathtofile%) is not read/writeable!
10:23:08 🚨 Can't connect to the database.
10:23:08 🚨 SQLITE_CANTOPEN: unable to open database file 
Error: SQLITE_CANTOPEN: unable to open database file`

I followed the instructions on https://docs.directus.io/getting-started/installation/iis/ and set the folder permissions to full control for the group IIS_IUSRS.

Any advice is appreciated. Thank you

**Comment by rijkvanzanten** [comment]
What ENV configuration are you using @hagenderouen? 

@paulboudewijn any insights? I'm not familiar with IIS at all.

**Reply by hagenderouen** [reply]
The default ENV that came with `npm init directus-project example-project`

**Comment by hagenderouen** [comment]
Solved by changing DB_FILENAME env variable to relative path

**Comment by rowild** [comment]
This happened to me, too, when changing from npm to pnpm. @hagenderouen answers worked great for me. Thank you!
